docs/tutorials/anomaly-detection.rst

```diff
@@ -0,0 +1,614 @@
+
+   Once created, the source table tab should look like the following:
+
+   .. image:: /images/tutorials/anomaly-detection/CPU_IN_source.png
+      :alt: Source table tab with ``CPU_IN`` table defined
+
+   You can check the if the source definition matches the data in the topic by clicking on the triangle next to **Run**, you should see the populated data
+
+   .. image:: /images/tutorials/anomaly-detection/cpu_in_table_preview.png
+      :alt: The Apache Flink source definition with SQL preview of the data
+
+
+
+5. Navigate to the **Add sink table** tab
+6. Create the sink table (named ``CPU_OUT_FILTER``), pointing to a new Apache Kafka® topic named ``cpu_load_stats_real_filter`` where the readings exceeding the ``80%`` threshold will land, by:
+
+   * Clicking on the **Add your first sink table**
+   * Selecting ``Aiven for Apache Kafka - demo-kafka`` as `Integrated service`
+   * Pasting the following SQL:
+
+     .. literalinclude:: /code/products/flink/basic_cpu-out-filter_table.md
+         :language: sql
+
+   Once created, the sink table tab should look like the following:
+
+   .. image:: /images/tutorials/anomaly-detection/CPU_OUT_target.png
+      :alt: Sink table tab with ``CPU_OUT`` table defined
+
+
+7. Navigate to the **Create statement** tab.
+8. Enter the following as the transformation SQL statement, taking data from the ``CPU_IN`` table and pushing the samples over the ``80%`` threshold to ``CPU_OUT_FILTER``:
+
+   .. literalinclude:: /code/products/flink/basic_job.md
+      :language: sql
+
+   If you're curious, you can preview the output of the transformation by clicking on the triangle next to the **Run** section, the *Create statement* window should be similar to the following image.
+
+   .. image:: /images/tutorials/anomaly-detection/filtering-preview.png
+      :alt: The Apache Flink data transformation with SQL preview of the data
+
+9. Click **Save and deploy later**
+10. Click **Create deployment**. 
+11. Accept the default deployment parameters and click on **Deploy without a savepoint**
+
+    .. image:: /images/tutorials/anomaly-detection/filtering-application-deployment.png
+        :alt: Detail of the new deployment screen showing the default version, savepoint and parallelism parameters
+
+12. The new application deployment status will show **Initializing** and then **Running: version 1**.
+
+Once the application is running, you should start to see messages indicating hosts with high CPU loads in the ``cpu_load_stats_real_filter`` topic of your ``demo-kafka`` Apache Kafka service.
+
+.. image:: /images/tutorials/anomaly-detection/filtering-topic-preview.png
+      :alt: The Apache Flink data transformation with SQL preview of the data
+
+
+.. Important::
+    
+    Congratulations! You created your first streaming anomaly detection pipeline!
+
+
+Create an advanced anomaly detection pipeline with windowing and threshold lookup
```

Comment:
Maybe better to name it "Adding improvements" or "Evolving" (I can't come up with a good name), a name that will indicate that the work will be on top of what is created already, and not a separate new tutorial